Azure/azure-cli
# [SQL] az sql db replica create: Add --secondary-type argument

**Description**
This change adds --secondary-type parameter to az sql db replica create command.  This allows caller to specify which type of secondary they want to be created.

**Testing Guide**
Test updates included with change

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).
